docs/Unit-tests.rst

```diff
@@ -182,6 +182,20 @@ To only run the subgroup of tests for ``filetools``::
 
     python -m test.framework.filetools
 
+Subsuites can be filtered on method names. Simply add the string to filter on to the test command. For example, to run
```

Comment:
`Since EasyBuild v2.8.2, tests can be filtered by name.`

(don't mention subsuites, since the filtering also works for the full suite :))